packages/now-bash/index.js

```diff
@@ -24,10 +33,23 @@ exports.build = async ({
   await download(files, srcDir);
 
   const configEnv = Object.keys(config).reduce((o, v) => {
-    o[`IMPORT_${snakeCase(v).toUpperCase()}`] = config[v]; // eslint-disable-line no-param-reassign
+    const name = snakeCase(v).toUpperCase();
+
+    if (allowedConfigImports.has(name)) {
+      o[`IMPORT_${name}`] = config[v]; // eslint-disable-line no-param-reassign
+    }
+
     return o;
   }, {});
 
+  if (config && config.imports) {
+    Object.keys(config).forEach((key) => {
```

Comment:
```suggestion
    Object.keys(config.import).forEach((key) => {
```

I believe this needs to be ☝️

Reply:
correct 👍
though, `imports` instead of `import` :D